wiki/markdown/mapping-rules-to-aggregations.md

```diff
@@ -0,0 +1,10 @@
+When writing subscriptions it is important that you know the structure of the aggregated object. In this section we present how example aggregated objects look like in relation to our existing rules.
+
+
+[**Artifact Rules**](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/ArtifactRules.json) will result in an [**Artifact Object**](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/test/resources/AggregatedDocumentInternalCompositionLatestIT.json) based on following [**events**](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/test/resources/ArtifactFlowTestEvents.json) from message bus and following historical [**events**](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/test/resources/upStreamInput.json) from event repository.
+
+
+[**Source Change Rules**](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/SourceChangeObjectRules.json) will result in [**Source Change Object**](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/test/resources/aggregatedSourceChangeObject.json) based on following [**events**](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/test/resources/TestSourceChangeObject.json) from message bus.
```

Comment:
I just wonder why the resource file is called TestSrouceChangeObject.json when it contains events... But I guess this is for another PR because it may affect tests if we rename it?